mlrun/runtimes/nuclio/application/application.py

```diff
@@ -509,6 +509,13 @@ def invoke(
 
         if not method:
             method = "POST" if body else "GET"
+
+        # Determine the correct keyword argument for the body
+        if isinstance(body, dict):
+            http_client_kwargs["json"] = body
+        elif isinstance(body, (str, bytes)):
+            http_client_kwargs["data"] = body
```

Comment:
We should add the `body` argument to the api_gateway's `invoke` function, and this should be moved there.
Then body will be supported both in application and api gateway in general

Reply:
@TomerShor api gw already supports data/json - whatever user sets in kwargs, it's just a matter of decision, but I think both options are fine, so will move to gws